docs/implementation/lift-inference.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: docs
+title: Lift Inference
+---
+
+# Lift Inference
+
+Programs in the Effekt surface language are translated to the underlying Core language.
+In order to efficiently compile programs in Core, the overhead incurred by using
+effect handlers should be minimized. In particular, the search for the correct handler
+at runtime should be avoided. This can be facilitated by making so-called lifting
+information explicit and using this information for a translation to iterated CPS.
```

Comment:
That's correct, but maybe mention that Core already is in "capability-passing style"?